.github/workflows/deploy_nightly.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       # not flagged as the latest release, which means that people will not get this
       # version of the package unless requested explicitly
       - name: publish nightly release
-        run: yarn changeset publish --tag nightly
+        run: yarn workspaces foreach -v --no-private npm publish --tag nightly
```

Comment:
The use of `--exclude=root` varies. Not needed here?

Reply:
Yeah not really, since the root package is private